iree_tests/README.md

```diff
@@ -1,5 +1,39 @@
 # IREE Tests
 
+## Quickstart instructions
+
+```
+#!/bin/bash
+set -e
+
+cd ~/SHARK-TestSuite/
+
+# Create and activate a virtual environment
+python3 -m venv .venv
+source .venv/bin/activate
+
+# Install required packages
+pip install -r iree_tests/requirements.txt
+pip install ./common_tools/
+
+# Optional: Install IREE compiler and runtime if you don't want to use a local copy
+# pip install --find-links https://iree.dev/pip-release-links.html iree-compiler iree-runtime --upgrade
+
+# Install Git LFS
+git lfs install
+
+# Optional: Download large remote files required for certain tests
+# python download_remote_files.py
+
+# Run a simple test to verify setup
+pytest iree_tests/simple
+
+echo "Setup complete! You can now run IREE tests."
+echo "Example: pytest iree_tests -n auto"
```

Comment:
We rarely ever run all tests with the same config file. Consider the simple case of CPU:

* https://github.com/nod-ai/SHARK-TestSuite/blob/main/iree_tests/configs/onnx_cpu_llvm_sync.json
* https://github.com/nod-ai/SHARK-TestSuite/blob/main/iree_tests/configs/models_cpu_llvm_task.json

The ONNX tests run single threaded (`--device=local-sync') and have lists of XFAIL'd ONNX tests.
The model tests run multithreaded (`--device=local-task`), enable some baseline optimizations (`--iree-llvmcpu-target-cpu-features=host`), and have lists of XFAIL'd model tests.

I'd put a command here that is expected to work out of the box. Maybe something more like `pytest iree_tests -n auto -k "test_sub_"` (see "Run a filtered subset of tests" further down in this file) or `pytest iree_tests/onnx -n auto`.